WhosThat/WhosThat/BusinessLogic/PersonRecognition.cs

```
@@ -41,20 +43,17 @@ private PersonRecognition()
 
         public bool LoadTrainedFaces()
```

Perhaps change it to void, as bool, in this case, is not very useful.

---

then we will need to throw an exception, to signal that the face photos were not found